Model/Mesh Statistics/MeshStatistics.h

```diff
@@ -0,0 +1,32 @@
+//
+// Created by Panda on 29.03.2017.
+//
+
+#ifndef  MeshStatistic_H_
+#define  MeshStatistic_H_
+
+#include "../Common/CommonMeshRepresentation.h"
+
+using namespace MeshIS::Model::Common;
+
+namespace MeshIS
+{
+    namespace Model
+    {
+        class MeshStatistics
+        {
+        public:
+            MeshStatistics() {};
```

Comment:
I intended to keep my branch as lightweight as possible and I also thought that when (MORE LIKE IF) you merge it into the master there won't be any conflicts with those files. Should i add them back?